bokeh/server/configure.py

```diff
@@ -88,7 +88,7 @@ def configure_flask(config_argparse=None, config_file=None, config_dict=None):
 def register_blueprint():
     global registered
     if registered:
-        warnings.warn(
+        logging.warn(
```

Comment:
I just tried this:
 

    >>> import bokeh    
    >>> bokeh.logging.warn('asd')
    WARNING:root:asd          
    >>> import logging    
    >>> logging.warn('asd')
    WARNING:root:asd